.github/workflows/ci.yml

```diff
@@ -108,6 +108,7 @@ jobs:
       - run: npm run dry-run
         env:
             GLIF_TOKEN: ${{ secrets.GLIF_TOKEN }}
+            INFLUXDB_TOKEN: ${{ secrets.INFLUXDB_TOKEN }}
```

Comment:
I stored the token in GitHub Actions secrets. The token has a read-only access to the bucket `spark-evaluate`.